src/main/java/com/google/cloud/tools/maven/CloudSdkAppEngineFactory.java

```diff
@@ -241,6 +252,25 @@ public void onOutputLine(String line) {
     }
   }
 
+  /** A listener that redirects process output to a file. */
+  protected static class FileOutputLineListener implements ProcessOutputLineListener {
+
+    private final PrintStream logFilePrinter;
+
+    public FileOutputLineListener(File logFile) {
+      try {
+        logFilePrinter = new PrintStream(logFile, "UTF-8");
```

Comment:
I believe `PrintStream` should be closed after use.

Reply:
I'm not sure where I'd be able to close this (the gradle plugin uses the same class but doesn't close it either).

Reply:
Perhaps by attaching an exit listener?

Reply:
It also looks like PrintWriter wrapping a BufferedWriter might be preferred for appending strings to a file over PrintStream, but this should be fine since it's how the gradle plugin does it.

https://stackoverflow.com/questions/11372546/printstream-vs-printwriter